README.md

```diff
@@ -31,6 +31,7 @@ that people are likely to search for.
 | 11 | 0x800 | [Cilium][cilium] |
 | 14 | 0x4000 | [Kubernetes][k8s] |
 | 15 | 0x8000 | [Kubernetes][k8s] |
+| 16-31 | 0xFFFF0000 | [Calico][cal] |
```

Comment:
Hmm, I'm not sure if we want to do N-M formatting, or break out each bit into its own column. I started with the latter, but if there's lots of software that uses many bits, maybe it's not worth it...

I'll take this as-is for now and experiment with different formatting.

Reply:
Yeah thinking this over I agree that having a line for each bit would make it easier to spot conflicts as there would be more than one service in a single line.

Reply:
Yeah, but on the other hand having 16 lines that just say "Calico" is really noisy as well... I was hoping that most software would only use 1-2 bits each, so it wouldn't be too bad. Calico using fully 1/2 of the space makes that a pretty bad option, unfortunately :(